### ISSUE TITLE
can"t disabled

### ISSUE THREAD

**Issue by Kaiter-Plus** [antfu/vscode-file-nesting-config#102]
The function is still works when I disabled this extension

**Comment by leonardssh** [comment]
What do you mean it still works? Are you still getting that update dialog? Or are your still have file nesting?

**Comment by Kaiter-Plus** [comment]
Yes, the files are still nested, and I need to modify the configuration to disable it
```json
"explorer.experimental.fileNesting.enabled": true
```
👇
```json
"explorer.experimental.fileNesting.enabled": false
```

**Comment by leonardssh** [comment]
Disabling the extension will not manipulate your config. 

File Nesting is a vscode feature. This extension only updates the pattern.

**Comment by Kaiter-Plus** [comment]
OK, thanks😋
